applications/app/controllers/CrosswordsController.scala

```diff
@@ -1,30 +1,21 @@
 package controllers
 
-import com.gu.contentapi.client.model.{Content => ApiContent, Crossword, Section => ApiSection}
-import common.{Edition, ExecutionContexts}
+import com.gu.contentapi.client.model.{Section => ApiSection}
+import common.ExecutionContexts
 import conf.{LiveContentApi, Static}
 import crosswords.{AccessibleCrosswordRows, CrosswordData, CrosswordPage, CrosswordSearchPage, CrosswordSvg}
 import model.{Cached, Cors, _}
 import org.joda.time.DateTime
 import play.api.data.Forms._
 import play.api.data._
-import play.api.mvc.{Action, Controller, RequestHeader, Result, _}
+import play.api.mvc.{Action, Controller, _}
 import services.IndexPage
+import services.ContentApiGetters.withCrossword
 
 import scala.concurrent.Future
 import scala.concurrent.duration._
 
 object CrosswordsController extends Controller with ExecutionContexts {
-  protected def withCrossword(crosswordType: String, id: Int)(f: (Crossword, ApiContent) => Result)(implicit request: RequestHeader): Future[Result] = {
```

Comment:
Makes sense. Although you could have still done it straight off the controller if you needed!